airflow/providers/fab/auth_manager/cli_commands/user_command.py

```diff
@@ -212,10 +212,10 @@ def users_import(args):
 
     users_created, users_updated = _import_users(users_list)
     if users_created:
-        print("Created the following users:\n\t{}".format("\n\t".join(users_created)))
+        print(f"Created the following users:\n\t\n\t{users_created}")
```

Comment:
How is this change related to the PR?

Reply:
Not really related, but some how the ruff format will change it every time I run the pre-commit. 

I could revert it if it makes it more clear.

Reply:
Please do. If this change is needed then it should be a seperated PR